lib/instances/GuiObject.lua

```diff
@@ -0,0 +1,94 @@
+local BaseInstance = import("./BaseInstance")
+local Color3 = import("../types/Color3")
+local InstanceProperty = import("../InstanceProperty")
+local Signal = import("../Signal")
+local UDim2 = import("../types/UDim2")
+local Vector2 = import("../types/Vector2")
+local GuiObject = BaseInstance:extend("GuiObject")
+
+GuiObject.properties.Active = InstanceProperty.typed("boolean", {
+	getDefault = function()
+		return true
+	end,
+})
+
+
+GuiObject.properties.AnchorPoint = InstanceProperty.typed("Vector2", {
+	getDefault = function()
+		return Vector2.new()
+	end,
+})
+
+GuiObject.properties.AutoButtonColor = InstanceProperty.typed("boolean", {
+	getDefault = function()
+		return false
+	end,
+})
+
+GuiObject.properties.BackgroundColor3 = InstanceProperty.typed("Color3", {
+	getDefault = function()
+		return Color3.new()
+	end,
+})
+
+GuiObject.properties.BackgroundTransparency = InstanceProperty.typed("number", {
+	getDefault = function()
+		return 0
+	end,
+})
+
+GuiObject.properties.BorderSizePixel = InstanceProperty.typed("number", {
+	getDefault = function()
+		return 0
+	end,
+})
+
+GuiObject.properties.ClipsDescendants = InstanceProperty.typed("boolean", {
+	getDefault = function()
+		return false
+	end,
+})
+
+GuiObject.properties.Frame = InstanceProperty.typed("UDim2", {
+	getDefault = function()
+		return UDim2.new()
+	end,
+})
```

Comment:
Frame isn't a _property_, is it?

Reply:
I don't even know where to begin on why I put this here